src/components/atoms/Button/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { IButtonProps } from "../../../utils/interfaces";
+import { PrimaryButton, SecondaryButton, GhostButton } from "./styles";
+
+function Button({ theme, buttonStyle, children, ...rest }: IButtonProps) {
+  const renderButton = () => {
+    switch (buttonStyle) {
+      case "primary":
```

Comment:
You can remove case "primary" if you use the default Primary button as a default

Reply:
I changed the default case instead of the primary case for explicit type reason.